.dev_scripts/test_regression_txt2img_dream_v1_4.sh

```diff
@@ -0,0 +1,19 @@
+# generate an image
+PROMPT_FILE=".dev_scripts/sample_command.txt"
+OUT_DIR="outputs/img-samples/test_regression_txt2img_v1_4"
+SAMPLES_DIR=${OUT_DIR}
+python scripts/dream.py \
+    --from_file ${PROMPT_FILE} \
+    --outdir ${OUT_DIR} \
+    --sampler plms
```

Comment:
Think you should add --full_precision here.

Reply:
Thank you for pointing it out.